src/components/navBar/navBar.css

```diff
@@ -0,0 +1,96 @@
+.navBar-container {
+    display: flex;
+    flex-direction: row;
+    width: 1440px;
+    height: 53px;
+    padding: 14px 123px 5px;
+    background-color: var(--white);
```

Comment:
Note that var(--white) will have the same effect as writing white.
If you want to use variables to hold your colors I would recommend learning and using SASS to write all of your stylings, that way you can create variables that hold colors, fonts and more and only reference them when needed.
**Reference for SASS basics** - https://sass-lang.com/guide
**How to write SASS with React** - https://scotch.io/starters/react/adding-sass-to-create-react-app-applications